modules/distr-tracing-rn-technology-preview.adoc

```diff
@@ -17,6 +17,35 @@ Technology Preview features are not supported with Red Hat production service le
 These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
 ====
 
+== {DTProductName} 2.1.0 Technology Preview
+
+This release introduces a breaking change to how to configure certificates in the OpenTelemetry custom resource file.  In the new version, the `ca_file` moves under `tls` in the custom resource, as shown in the two examples below.
```

Comment:
Looks like there may be a double space between the first and second sentence. 

IBM SG: Don't use "below" to indicate a relative location in a document, as in "the information below". Use "the information later in this section". Alternatively, you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."

Reply:
Fixed.